pom.xml

```diff
@@ -100,8 +100,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.5.1</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
```

Comment:
I guess with retrolambda and now the Android jack compiler, targeting Java 8 is going to be possible even on older versions of Android so this is less a concern.  I'm good with moving to 8.